src/main/java/seedu/duke/storage/JsonUtil.java

```diff
@@ -27,37 +27,48 @@ public class JsonUtil {
             .enable(SerializationFeature.INDENT_OUTPUT) // Readable format // print
             .findAndRegisterModules(); // Automatically register additional modules (future use);
 
+    public static void saveToFile(String filePath) {
+        try {
+            objectMapper.writeValue(new File(filePath), new Hospital());
+        } catch (IOException e) {
+            // TODO: Update error handler (yes I am lazy)
```

Comment:
Try to implement this as soon as possible as saving files is one of our core features.
Perhaps this could work for the error handler:
logger.log(Level.SEVERE, "Failed to save data to file: {0}", e.getMessage());
throw new RuntimeException("Failed to save hospital data", e);